Eshop.WebApi/Program.cs

```diff
@@ -18,6 +25,28 @@
 
 var app = builder.Build();
 
+//EF
+CreateDbIfNotExists(app);
+
+static void CreateDbIfNotExists(IHost host)
+{
+    using (var scope = host.Services.CreateScope())
+    {
+        var services = scope.ServiceProvider;
+        try
+        {
+            var context = services.GetRequiredService<EshopDbContext>();
+            context.Database.EnsureCreated();
+            var test = context.Categories.ToList();
+        }
+        catch (Exception) 
+        {
+            throw;
+        }
+
```

Comment:
you are always having empty not used spaces, formating is diligence of the programmer that he takes care

Reply:
fixed